config/pmd.xml

```diff
@@ -175,7 +175,7 @@
     <!-- Too many false positives. -->
     <exclude name="DataflowAnomalyAnalysis"/>
     <!-- This rule does not have any option, unreasonable to use. -->
-    <exclude name="MissingBreakInSwitch"/>
```

Comment:
The rule was renamed to [ImplicitSwitchFallThrough](https://github.com/pmd/pmd/releases#renamed-rules)